docs/formatters.md

```diff
@@ -39,7 +39,7 @@ Html can be generated sever-side the same way, just remember to include (or embe
 Console
 -------
 
-Node.js colored output (thanks to [chalk](https://github.com/sindresorhus/chalk)!).
```

Comment:
@benjamine I can delete this if you want, I simply replaced every instance of chalk.